src/aideck/embeddings/azure_openai_provider.py

```
@@ -0,0 +1,449 @@
+                except:
+                    pass
+            
+            await asyncio.sleep(min(wait_time, 300))
+            raise
+            
+        except openai.AuthenticationError as e:
+            if self.config.use_managed_identity:
+                try:
+                    await self._get_azure_token()
+                    raise
+                except:
```

Ruff (0.13.1)

Do not use bare `except`

(E722)